src/main/java/com/iexec/core/configuration/WorkerConfiguration.java

```diff
@@ -11,4 +11,8 @@
 
     @Value("${workers.askForReplicatePeriod}")
     private long askForReplicatePeriod;
+
+    @Value("${workers.workerVersionRequired}")
+    private String workerVersionRequired;
+
```

Comment:
maybe requiredWorkerVersion would be clearer (?)

Reply:
Yes, I wasn't sure !